recipes/vim/vim.inc

```diff
@@ -0,0 +1,65 @@
+DESCRIPTION = "Vi IMproved"
+LICENSE = "Vim License (GPL-compatible)" 
```

Comment:
Should simply be
LICENSE = "Vim"
(search for vim on https://spdx.org/licenses/)